lib/app-express.js

```diff
@@ -16,6 +37,48 @@ module.exports = function (app) {
     return next();
   });
 
+  app.get('/hostedOneClick', (req, res) => {
+    // feature not active--send you to the homepage
+    console.log('oneClick requested');
+    const template = req.query.template;
+    if (!allowHostedOneClick){
+      console.log('OneClick deploy not allowed');
+      res.render('pages/customError', {
```

Comment:
Instead of another error page, what if we add the custom error support into the main error page?

Reply:
Yeah--I was trying not to make huge changes to your existing stuff, but I agree that makes more sense.  Yours was moving towards extraneous pages for known errors like "not a hub".